deploy.sh

```diff
@@ -25,3 +26,7 @@ cf zero-downtime-push $APP_NAME -f $MANIFEST
 # TODO: Restore zero-downtime push after https://github.com/concourse/autopilot/issues/23 is resolved
 # cf zero-downtime-push $WORKER_NAME -f $MANIFEST
 cf push $WORKER_NAME -f $MANIFEST
+
+if [ $SPACE = 'prod' ]; then
```

Comment:
Since we'd normally track this in the manifest, could you add a comment here? Also, it sounds like cloud.gov is going to switch to billing by memory used rather than memory allocated to the quota (maybe quotas will go away too!), so I think we can mark this as a temporary workaround.